lib/ae_mdw/db/model.ex

```diff
@@ -102,15 +102,22 @@ defmodule AeMdw.Db.Model do
 
   # auction bid:
   #     index = {plain_name, {block_index, txi}, expire_height = height, owner = pk, prev_bids = []}
-  @auction_bid_defaults [index: {nil, {{nil, nil}, nil}, nil, nil, nil}, unused: nil]
+  @auction_bid_defaults [
+    index: nil,
+    block_index_txi: nil,
+    expire_height: nil,
+    owner: nil,
+    bids: []
+  ]
   defrecord :auction_bid, @auction_bid_defaults
 
-  @type auction_bid_key ::
-          {String.t(), {Blocks.block_index(), Txs.txi()}, Blocks.height(), Db.pubkey(), list()}
   @type auction_bid ::
           record(:auction_bid,
-            index: auction_bid_key(),
-            unused: nil
+            index: Names.plain_name(),
+            block_index_txi: Blocks.block_index_txi(),
```

Comment:
I think multiple bids should be allowed to a name.

Reply:
The history of bids is kept on the `bids: [Blocks.block_index_txi()]` but only one AuctionBid can exist per name

Reply:
Source: https://github.com/aeternity/ae_mdw/blob/master/lib/ae_mdw/db/mutations/name_claim_mutation.ex#L153

Reply:
Maybe it could keep the multiple bidders. It might be useful for https://github.com/aeternity/ae_mdw/issues/131

Reply:
That could be a useful feature, I would leave it out of the scope of this PR since it will require a bigger migration to obtain that information, though. Good thing is that the PR is already referenced by your previous comment